src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+nav {
+  height: 60px;
+  margin: 0;
+}
+
+.nav-links {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  height: 100%;
+}
+
+.nav-item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
```

Comment:
Be careful with the use of the `margin` property. It's generally a good practice to add only top or only bottom margin, but not both. Vertical margins can be collapsed in some cases. If you need space on both sides, consider using padding instead.